src/components/toggleSwitch/index.tsx

```diff
@@ -21,9 +21,11 @@ export const Toggle: FC<Props> = ({
   }
 
   return (
-    <ToggleWrapper isChecked={isChecked}>
-      <ToggleInput {...others} type='checkbox' onChange={handleChange} checked={isChecked} />
-      {isShowSpan && <ToggleSpan isChecked={isChecked}>{isChecked ? 'on' : 'off'}</ToggleSpan>}
-    </ToggleWrapper>
+    <ToggleSwitchWrapper isChecked={isChecked}>
+      <ToggleSwitchInput {...others} type='checkbox' onChange={handleChange} checked={isChecked} />
+      {isShowSpan && (
+        <ToggleSwitchSpan isChecked={isChecked}>{isChecked ? 'on' : 'off'}</ToggleSwitchSpan>
```

Comment:
Naming with adding to it the tag names isn't correct, 
it should be descriptive for other devs also, 
so they can assume what that element expected to do.

Reply:
corrected